sources/europe/gb/OSOpenMapLocal.geojson

```diff
@@ -1,12 +1,11 @@
 {
     "type": "Feature",
     "properties": {
-        "id": "OS-OpenMap_Local_2024_10",
-        "name": "OS OpenMap Local (October 2024)",
-        "url": "https://{switch:a,b,c}.os.openstreetmap.org/layer/gb_os_om_local_2024_10/{zoom}/{x}/{y}.png",
+        "id": "OS-OpenMap_Local_latest",
+        "name": "OS OpenMap Local",
+        "url": "https://{switch:a,b,c}.os.openstreetmap.org/om-local/{zoom}/{x}/{y}.png",
```

Comment:
```suggestion
        "url": "https://os.openstreetmap.org/om-local/{zoom}/{x}/{y}.png",
```
In the past sub domains offered enhanced performance but this is no longer true, so let's remove them here (ref https://operations.osmfoundation.org/policies/tiles/)